models/contact.py

```diff
@@ -0,0 +1,9 @@
+import mongoengine as me
+
+class Contact(me.Document):
+    id_organisme = me.ObjectIdField(required=True)
```

Comment:
it seems to be a reference fied, use https://docs.mongoengine.org/guide/defining-documents.html#reference-fields insted of ObjectIdField